evalml/automl/engine/dask_engine.py

```diff
@@ -159,6 +163,7 @@ def submit_scoring_job(self, automl_config, pipeline, X, y, objectives):
         X_schema = X.ww.schema
         y_schema = y.ww.schema
         X, y = self.send_data_to_cluster(X, y)
+        X_train, y_train = self.send_data_to_cluster(X_train, y_train)
```

Comment:
Just for my own curiosity: theoretically, if `send_data_to_cluster` supported more arguments, this could have been combined with the line above, right? 🤔